src/Reader/Feed/Rss.php

```diff
@@ -474,6 +474,16 @@ public function getLanguage()
         return $this->data['language'];
     }
 
+    /**
+     * Get a link to the source website. This is not implemented in RSS.
+     *
+     * @return void
+     */
+    public function getBaseUrl()
```

Comment:
Please only focus on adding psalm instead of adding features like this.
We should not add new methods to interfaces. If there are issues with psalm, just add these to the baseline.

Would love to see this as a separated PR. Maybe we will add it to the next release tho.